docs/meshstack.openshift.index.md

```diff
@@ -15,7 +15,12 @@ To enable integration with OpenShift, operators deploy and configure the meshSta
 
 ### OpenShift Versions
 
-meshStack currently supports OpenShift version 3.9+ as either Open-Source (OKD) or OpenShift Enterprise variants.
+meshStack currently supports OpenShift version 3.9+ and 4.x as either Open-Source (OKD) or OpenShift Enterprise variants.
+
+In general meshStack supports all OpenShift versions that can provide the resources listed for the required [service accounts](meshstack.openshift.index.md#meshstack-service-accounts). These resources are consumed with the following versions. So as long as the following endpoints are supported in the OpenShift version, meshStack should be able to work with it:
+
+* /api/v1
```

Comment:
d: this should also link to the K8s APIs we use (link to general kubernetes docs). 

d: maybe replace the exact APIs used with a link to the cluster role YAML which lists them as well in the integration section of the docs. "meshStack supports all Kubernetes distributions offering the following API versions"...

Reply:
I actually linked the cluster roles above. I changed the sentence a bit to make it clearer. It is just that e.g. for namespaces no version is provided of the API. So we have to add it here too.